src/components/claims/claim-details/components/ClaimPayment.tsx

```diff
@@ -48,16 +47,14 @@ export interface PaymentFormData {
   message?: string
 }
 
-const PaymentForm = styled(Form)({
-  marginTop: '1rem',
-})
+const PaymentForm = styled(Form)`
+  margin-top: 1rem;
+`
 
-const SubmitButton = withStyles({
-  root: {
-    marginTop: '1rem',
-    display: 'block',
-  },
-})(MuiButton)
+const SubmitButton = styled(MuiButton)`
+  margin-top: 1rem;
+  display: block;
+`
```

Comment:
Since it's a Material-UI button, you probably want to do something like:

```suggestion
const SubmitButton = styled(MuiButton)`
  && {
    margin-top: 1rem;
    display: block;  
  }
`
```

It might look good locally, but when you bundle in staging and production there's a risk it overrides the styles (which you prevent with this)